packages/commands/src/bitcoin/memo/deposit.ts

```diff
@@ -0,0 +1,85 @@
+import { Command, Option } from "commander";
+import { ethers } from "ethers";
+import { z } from "zod";
+
+import { memoDepositOptionsSchema } from "../../../../../types/bitcoin.types";
+import {
+  addCommonOptions,
+  broadcastBtcTransaction,
+  displayAndConfirmMemoTransaction,
+  fetchUtxos,
+  setupBitcoinKeyPair,
+} from "../../../../../utils/bitcoin.command.helpers";
+import {
+  bitcoinMakeTransactionWithMemo,
+  getDepositFee,
+} from "../../../../../utils/bitcoinMemo.helpers";
+import { validateAndParseSchema } from "../../../../../utils/validateAndParseSchema";
+
+type DepositOptions = z.infer<typeof memoDepositOptionsSchema>;
+
+const main = async (options: DepositOptions) => {
+  const { key, address } = setupBitcoinKeyPair(
+    options.privateKey,
+    options.name
+  );
+  const utxos = await fetchUtxos(address, options.bitcoinApi);
+
+  const receiver = options.receiver?.startsWith("0x")
+    ? options.receiver.slice(2)
+    : options.receiver;
+
+  const memo = receiver || "";
+
+  const amount = Number(ethers.parseUnits(options.amount, 8));
```

Comment:
Shouldn't we use the `parseAmount` helper located on utils/bitcoin.command.helpers.ts:242?

Reply:
Done